prow/gerrit/client/client.go

```diff
@@ -204,6 +204,36 @@ func NewClient(instances map[string]map[string]*config.GerritQueryFilter) (*Clie
 	return c, nil
 }
 
+func (c *Client) ApplyGlobalConfig(cfg config.Getter, lastSyncTracker *SyncTime, cookiefilePath, tokenPathOverride string, additionalFunc func()) {
+	c.applyGlobalConfigOnce(cfg, lastSyncTracker, cookiefilePath, tokenPathOverride, additionalFunc)
+
+	go func() {
+		for {
+			c.applyGlobalConfigOnce(cfg, lastSyncTracker, cookiefilePath, tokenPathOverride, additionalFunc)
+			// No need to spin constantly, give it a break. It's ok that config change has one second delay.
+			time.Sleep(time.Second)
+		}
+	}()
+}
+
+func (c *Client) applyGlobalConfigOnce(cfg config.Getter, lastSyncTracker *SyncTime, cookiefilePath, tokenPathOverride string, additionalFunc func()) {
```

Comment:
Looks like `additionalFunc` param is not used in the function body. Remove?

Reply:
it's actually used, added